Modules/Sources/Yosemite/Tools/POS/PointOfSaleSettingsService.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import Networking
+import Storage
+
+public protocol PointOfSaleSettingsServiceProtocol {
+    var siteID: Int64 { get }
+    func retrievePointOfSaleSettings() async throws -> [SiteSetting]
+}
+
+public final class PointOfSaleSettingsService: PointOfSaleSettingsServiceProtocol {
+    public private(set) var siteID: Int64
```

Comment:
nit: this could be let?
```suggestion
    public let siteID: Int64
```

Reply:
Adjusted https://github.com/woocommerce/woocommerce-ios/pull/16034/commits/61728b25ca9d62af535b9f472303bd792f96f711